<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogGroupEditor</class>
 <widget class="QDialog" name="DialogGroupEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabZone">
      <attribute name="title">
       <string>ZoneEditor</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Zone Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="editZoneName"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnZoneAdd">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>294</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Zone List</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" rowspan="2">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>81</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Groups In</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3" rowspan="5">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>169</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Groups Out</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" rowspan="8">
           <widget class="QListView" name="listViewZone"/>
          </item>
          <item row="1" column="2" rowspan="8">
           <widget class="QListView" name="listViewGroupIn"/>
          </item>
          <item row="1" column="4" rowspan="8">
           <widget class="QListView" name="listViewGroupOut"/>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Old Name</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="editZoneNameOld"/>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>New Name</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="editZoneNameNew"/>
          </item>
          <item row="5" column="3">
           <widget class="QPushButton" name="btnGroupOut">
            <property name="text">
             <string>&gt;&gt;</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QPushButton" name="btnZoneModify">
            <property name="text">
             <string>Modify</string>
            </property>
           </widget>
          </item>
          <item row="6" column="3">
           <widget class="QPushButton" name="btnGroupIn">
            <property name="text">
             <string>&lt;&lt;</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QPushButton" name="btnZoneDelete">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
          <item row="7" column="3" rowspan="2">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>154</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="8" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>111</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabGroup">
      <attribute name="title">
       <string>GroupEditor</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Group Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="editGroupName"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnGroupAdd">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>294</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Group List</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" rowspan="2">
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>81</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Device In</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3" rowspan="5">
           <spacer name="verticalSpacer_6">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>169</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Device Out</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" rowspan="8">
           <widget class="QListView" name="listViewGroup"/>
          </item>
          <item row="1" column="2" rowspan="8">
           <widget class="QListView" name="listViewDeviceIn"/>
          </item>
          <item row="1" column="4" rowspan="8">
           <widget class="QListView" name="listViewDeviceOut"/>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Old Name</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="editGroupNameOld"/>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>New Name</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="editGroupNameNew"/>
          </item>
          <item row="5" column="3">
           <widget class="QPushButton" name="btnDeviceOut">
            <property name="text">
             <string>&gt;&gt;</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QPushButton" name="btnGroupModify">
            <property name="text">
             <string>Modify</string>
            </property>
           </widget>
          </item>
          <item row="6" column="3">
           <widget class="QPushButton" name="btnDeviceIn">
            <property name="text">
             <string>&lt;&lt;</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QPushButton" name="btnGroupDelete">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
          <item row="7" column="3" rowspan="2">
           <spacer name="verticalSpacer_7">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>154</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="8" column="1">
           <spacer name="verticalSpacer_8">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>111</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
